src/components/SingleComment.jsx

```diff
@@ -14,17 +19,44 @@ const SingleComment = ({ comment }) => {
     });
   }, []);
 
+  const handleDelete = () => {
+    if (user.username === comment.author) {
+      setIsDeleting(true);
+      deleteComment(comment.comment_id)
+        .then(() => {
+          updateComments();
+          setDeleteSuccess(true);
+          setTimeout(() => {
+            setDeleteSuccess(false);
```

Comment:
Need to properly handle errors here - if I trigger a bad request (i.e. send a random string instead of the comment id to deleteComment) I get an uncaught error and no proper feedback on the screen, the comment just stays there